server.go

```diff
@@ -2432,9 +2438,13 @@ func newServer(listenAddrs []string, db database.DB, chainParams *chaincfg.Param
 					activeNetParams.DefaultPort && tries < 50 {
 					continue
 				}
-				return addrmgr.NetAddressKey(addr.NetAddress()), nil
+
+				addrString := addrmgr.NetAddressKey(addr.NetAddress())
+
```

Comment:
Super minor nit: This blank line probably shouldn't be here since it's grouped with the return in terms of logic.